addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -176,20 +176,14 @@
                     </div>
                     <ul t-if="aside_slide._has_additional_resources() or aside_slide.question_ids" class="o_wslides_lesson_aside_list_links list-group mb-1 list-unstyled fw-light">
                         <t t-if="can_access_channel">
-                            <t t-foreach="aside_slide.slide_resource_ids" t-as="resource" t-if="resource.resource_type == 'url'">
-                                <li class="ps-3">
-                                    <a t-if="can_access" t-att-href="resource.link" target="new" class="text-decoration-none small">
-                                        <i class="fa fa-link"/><span t-field="resource.name"/>
-                                    </a>
-                                </li>
-                            </t>
-                            <div class="o_wslides_js_course_join ps-3" t-if="aside_slide._has_additional_resources()">
-                                <li t-foreach="aside_slide.slide_resource_ids" t-as="resource" t-if="resource.resource_type == 'file'">
-                                    <a t-att-href="resource.download_url" class="text-decoration-none small">
-                                        <i class="fa fa-download"/><span t-field="resource.name"/>
-                                    </a>
-                                </li>
-                            </div>
+                            <li class="ps-3" t-foreach="aside_slide.slide_resource_ids" t-as="resource">
+                                <a t-if="can_access and resource.resource_type == 'url'" t-att-href="resource.link" target="new" class="text-decoration-none small">
+                                    <i class="fa fa-link"/><span t-field="resource.name"/>
+                                </a>
+                                <a t-else="" t-att-href="resource.download_url" class="text-decoration-none small">
```

Comment:
what happen if we can not access a resource of type url ? (before it was not visible I think, and now it will be downloadable)

Reply:
Hello @std-odoo ,
My bad we could probably use `t-elif="resource.resource_type == 'file'"` for that.
What's your thought?
Thanks

Reply:
we should keep the same behavior as before imo :)